apps/pom.xml

```diff
@@ -233,10 +233,12 @@
 					Hibernate is used for that in our projects.) -->
 				<artifactId>hibernate-jpa-2.1-api</artifactId>
 				<groupId>org.hibernate.javax.persistence</groupId>
-				<version>1.0.0.Final</version>
+				<version>1.0.2.Final</version>
 			</dependency>
+
+			<!-- Hibernate version not latest because entity manager does not have a 6.1 to upgrade to-->
```

Comment:
I think we can get rid of this comment in favor of one on the version itself

Reply:
Updated